examples/zone-printer/README.md

```diff
@@ -56,6 +56,22 @@ used by `kubemci` to provision the multi-cluster Ingress.
     gcloud components install kubectl
     ```
 
+4. Install **`kubemci`**:
+
+    You can install `kubemci` by running:
+
+    ```shell
+    wget https://storage.googleapis.com/kubemci-release/release/latest/bin/linux/amd64/kubemci
```

Comment:
Note that we have 2 binaries:
Linux, amd64: https://storage.googleapis.com/kubemci-release/release/latest/bin/linux/amd64/kubemci
darwin (MacOSX), amd64: https://storage.googleapis.com/kubemci-release/release/latest/bin/darwin/amd64/kubemci

You could also point to https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/releases if users want a specific version

For reference, look at "Download the kubemci command-line tool" section in https://cloud.google.com/kubernetes-engine/docs/how-to/kubemci

Reply:
updated